src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -16,8 +16,10 @@ export interface BoxAISidebarContextValues {
     cache: { encodedSession?: string | null; questions?: QuestionType[] };
     contentName: string;
     elementId: string;
+    isFeedbackEnabled: boolean;
```

Comment:
what happens after the feedback buttons work? in terms making any api calls. i'm not seeing a changes in the apis, does that mean it was already implemented?

Reply:
When user clicks feedback buttons the `resinAction` is being called, no API call are made.
We keep the feedback data in resin tracking.